### DISCUSSION TITLE
[Support]: No PTZ controls on full screen mode

### DISCUSSION THREAD

**Discussion by a24me** [blakeblackshear/frigate#13048] [Camera Support]
### Describe the problem you are having

There are no PTZ and auto tracking controls in full screen mode for PTZ cameras.

### Steps to reproduce

Switch a PTZ camera to full screen mode.

### Version

0.14.0-da913d8

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

n/a

**Comment by hawkeye217** [comment]
This is not a bug, this behavior is intentional to avoid obscuring the camera while in full screen. On desktops, you can still use the arrow keys on the keyboard to move the camera while in full screen, and if you enable the autotracking mode before going full screen, clicking/tapping on an area will still move the camera.

**Reply by Billy-** [reply]
We do already have a whole toolbar of buttons in full screen mode. Could we consider adding one button there, which then toggles PTZ etc. (like enabling microphone) controls?
